src/app/results/results.component.ts

```diff
@@ -81,7 +81,7 @@ export class ResultsComponent implements OnInit {
   }
 
   imageClick() {
-    this.getPresentPage(0);
+    this.getPresentPage(1);
     this.resultDisplay = 'images';
```

Comment:
I think `this.getPresentPage(0)` was added to solve pagination issues which we were facing earlier. Kindly explain.